base-ubuntu-2204/files/etc/ssh/sshd_config

```diff
@@ -0,0 +1,124 @@
+
+# This is the sshd server system-wide configuration file.  See
```

Comment:
@pan3793 Shall we remove all the commented-out lines?

Reply:
okay~ I believe nobody cares advanced ssh feature in this cases